test/framework/functional/elasticsearch.go

```diff
@@ -82,3 +83,69 @@ func (f *CollectorFunctionalFramework) GetLogsFromElasticSearchIndex(outputName
 	log.V(4).Info("Returning", "logs", results)
 	return results, err
 }
+
+func (f *CollectorFunctionalFramework) GetLogsFromElasticSearchWithBasicSecurity(outputName, outputLogType, username, password string, options ...Option) (results []string, err error) {
+	index, ok := ElasticIndex[outputLogType]
+	if !ok {
+		return []string{}, fmt.Errorf(fmt.Sprintf("can't find log of type %s", outputLogType))
+	}
+
+	port := "9200"
+	if found, o := OptionsInclude("port", options); found {
+		port = o.Value
+	}
+	err = wait.PollUntilContextTimeout(context.TODO(), defaultRetryInterval, maxDuration, true, func(cxt context.Context) (done bool, err error) {
+		cmd := `curl -u ` + `'` + username + `'` + `:` + `'` + password + `' ` + `-X GET "localhost:` + port + `/` + index + `/_search?pretty" -H 'Content-Type: application/json' -d'
+{
+	"query": {
+	"match_all": {}
+}
+}
+'`
+		fmt.Println(cmd)
```

Comment:
print statement.  replace with log statement

Reply:
I've removed this